src/Microsoft.XmlSerializer.Generator/src/Sgen.cs

```diff
@@ -73,7 +73,7 @@ private int Run(string[] args)
                     {
```

Comment:
> arg = arg.ToLower(CultureInfo.InvariantCulture);

I cannot add a comment at line 56, so I add here. We should not need line 56 anymore if you use `InvariantCultureIgnoreCase` in your string comparison in `ArgumentMatch` function.

Reply:
@zhenlan If we compare the strings using InvariantCultureIgnoreCase in ArgumentMatch function and delete line 56, there will be problems in line 123 where it compares if the string ends with ".dll" or ".exe".

Reply:
I believe EndsWith has overload to accept StringComparison argument too.